analyses/clone-phylogeny-analysis/06B-create-numbat-plots-multiple-samples.R

```diff
@@ -76,7 +77,7 @@ if (!dir.exists(numbat_plots_dir)) {
 
 #######################################################
 # Read metadata file and define `sample_name`
-metadata_file <- file.path(metadata_dir, "project_metadata.tsv") # metadata input file
+metadata_file <- file.path(metadata_dir, metadata_file) # metadata input file
```

Comment:
Same commenting regarding the `metadata_file` variable here and on line `83`.

```suggestion
project_metadata_file <- file.path(metadata_dir, metadata_file) # metadata input file
```

Reply:
Fixed here: 0f238c2d3a172b41d6d0fd3842f3f90d086f6874.